.github/workflows/changeset.yml

```diff
@@ -0,0 +1,21 @@
+name: Changeset
+
+on:
+  push:
+    branches:
+      - "main"
```

Comment:
Initially, thought to run workflow only if `.changeset/**` changes but then realized that `package.json` of packages may change too, causing changeset validation to fail. So now including all paths, although could still narrow down to `.changeset/**` and `packages/**/package.json`?